iceoryx_examples/icecubetray/config_and_run_icecubetray.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# ***NOTE***
+# This shell script is for Linux-based operating systems only.
+# If you're using e.g QNX refer to the manual on how to set up groups, users and permissions
+
+WORKSPACE=$(git rev-parse --show-toplevel)
+CONFIG="OFF"
+RUN="OFF"
+SESSION=icecubetray
+tmux="tmux -2 -q"
+
+while (( "$#" )); do
+  case "$1" in
+    config)
+        CONFIG="ON"
+        shift 1
+        ;;
+    run)
+        RUN="ON"
+        shift 1
+        ;;
+    *)
+        echo "Invalid argument '$1'"
+        exit -1
+        ;;
+  esac
+done
+
+if [ "$CONFIG" == "ON" ] ; then
+    # Create groups
+    sudo groupadd -f privileged
+    sudo groupadd -f unprivileged
+    sudo groupadd -f iceoryx
+
+    # Create users w/o homedir
+    sudo useradd -M perception
+    sudo useradd -M infotainment
+    sudo useradd -M roudi
+    sudo useradd -M notallowed
+
+    # Assign users to group and disable login
+    sudo usermod -L perception -a -G privileged
```

Comment:
You do not need to write usermod here you can pack everything in one command which makes this part less error prone. Just write:
```sh
usermod -M -G iceoryx,unprivileged -s /sbin/nologin infotainment
```

Some more infos:
1. every user is per default locked in a unix system. If you would like to unlock it you have to set a password with passwd which sets the field in /etc/shadow and unlocks the user
2. to deny the user to be used other then to run a specific application (deny login) we use `-s /sbin/nologin`. Otherwise you could always do stuff like: `su - infotainment` even when the user is locked, which we do not want.
3. After `-G` you can add a comma separated list of groups.

Reply:
Thanks for the explanation! Added